src/Guide/Api/Types.hs

```diff
@@ -106,6 +109,27 @@ data CategorySite route = CategorySite
            "group" Text
       :> Post '[JSON] (Uid Category)
 
+  , _setCategoryNotes :: route :-
+      Summary "Edit category's note"
+      :> ErrorResponse 404 "Category not found"
+      :> "category"
+      :> Capture "id" (Uid Category)
+      :> "notes"
+      :> ReqBody '[JSON] Text
+      :> Put '[JSON] NoContent
+
+  , _setCategoryInfo :: route :-
+      Summary "Set category's fields"
+      :> Description "Set category fields:\
```

Comment:
This description is not needed because the schema for `CCategoryInfoEdit` will already mention that.